cfme/infrastructure/virtual_machines.py

```diff
@@ -1360,6 +1361,15 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.lifecycle.item_select("Migrate this VM")
 
 
+@navigator.register(Vm, 'Publish')
+class VmPublish(CFMENavigateStep):
+    VIEW = PublishVmView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self, *args, **kwargs):
```

Comment:
Required: we don't need *args, **kwargs here, as we are not using them further in the step